scripts/zones/Garlaige_Citadel/mobs/Serket.lua

```diff
@@ -22,7 +22,7 @@ end
 
 entity.onMobDespawn = function(mob)
     UpdateNMSpawnPoint(mob:getID())
-    mob:setRespawnTime(math.random(75600, 86400)) -- 21 to 24 hours
+    mob:setRespawnTime(math.random(3600, 43200)) -- 1 to 12 hours
```

Comment:
I believe this is a 2014 change. We should keep note of this when merging with horizon

Reply:
Got a note about this in the #in-era-nm-discussion channel on Discord. Theres a few NMs I'll need to adjust in a module.